### PR TITLE
fix: Rollback the manual major version changes and change release command to increment the major versions

### PR DESCRIPTION
**Pull request by stocaaro**
#### Description of changes
fix: Rollback the manual major version changes and change release command to increment the major versions

#### Description of how you validated changes
Tested locally using verdaccio

Screen Shot 2022-11-08 at 2 38 37 PM

#### Checklist

- [x] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jimblanc — DISMISSED** [review]

File: packages/aws-amplify-react-native/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
 	},
 	"peerDependencies": {
 		"@react-native-picker/picker": ">=1.2",
-		"aws-amplify": "5.x.x",
+		"aws-amplify": "4.x.x",
```
Comment (elorzafe) [inline packages/aws-amplify-react-native/package.json:48]: I think lerna will fail to bump this one, I would keep it as `5.x.x`

**Review by elorzafe — DISMISSED** [review]
Looks good to me

Thanks @stocaaro

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
 		"publish:main": "lerna publish --canary --force-publish \"*\" --yes --dist-tag=unstable --preid=unstable --exact --no-verify-access",
 		"publish:next-major-version/5": "lerna publish --canary --force-publish \"*\" --yes --dist-tag=next --preid=next --exact --no-verify-access",
 		"publish:beta": "lerna publish --canary --force-publish \"*\" --yes --dist-tag=beta --preid=beta --exact --no-verify-access",
-		"publish:release": "lerna exec npm publish",
+		"publish:release": "lerna publish major --yes --dist-tag=next --message 'chore(release): Publish [ci skip]' --no-verify-access",
```
Comment (iartemiev) [inline package.json:27]: Needs `--exact`

**Review by iartemiev — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (iartemiev) [reply package.json:27]: And `--force-publish \"*\"`

**Review by iartemiev — APPROVED** [review]
👍🏻

**Review by jimblanc — APPROVED** [review]

**Review by elorzafe — APPROVED** [review]
LGTM

**Pull request merged**